http.py

```diff
@@ -114,8 +115,9 @@ def __init__(self, http_request):
         self.response_object = None
         self.request = None
         self.sock = None
-        self.CIPHERS = 'ADH-AES256-SHA:ECDHE-ECDSA-AES128-GCM-SHA256: \
-                ECDHE-RSA-AES128-GCM-SHA256:AES128-GCM-SHA256:AES128-SHA256:HIGH:'
+        self.CIPHERS = \
```

Comment:
The way it was before had the \ inside of the string which looked like

```
>>> self.CIPHERS
'ADH-AES256-SHA:ECDHE-ECDSA-AES128-GCM-SHA256:                 ECDHE-RSA-AES128-GCM-SHA256:AES128-GCM-SHA256:AES128-SHA256:HIGH:'
```

when evaluated which I assume is not correct? It's all supposed to be one string right?